xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/layout.less

```diff
@@ -302,6 +303,23 @@
   padding: @line-height-computed @grid-gutter-width;
 }
 
+#hierarchy_breadcrumb {
+  padding-bottom: 0;
+
+  .breadcrumb {
+    margin-bottom: 0;
+  }
+}
+
+/* Protect against nesting of the breadcrumb inside #mainContentArea, e.g., in AdminSheet and other containers that
+already take care of padding and shadow. */
+#mainContentArea #hierarchy_breadcrumb, #xwikieditor #hierarchy_breadcrumb {
+  padding-bottom: @line-height-computed;
+  /* Padding and shadow is already in the #mainContentArea element */
+  padding-top: 0;
+  box-shadow: none;
+}
+
```

Comment:
@manuelleduc I must admit that I don't understand how the changes from this commit are related to ticket XWIKI-21106, maybe an additional comment on the ticket itself with examples of what else needs to be handled besides what I described could help here...

Reply:
I made a temporary mistake while fixing a merge conflict. This is not part of the final commit, sorry.